cobrawap/pipeline/stage02_processing/Snakefile

```diff
@@ -13,7 +13,7 @@ def input_file(wildcards):
 
 def is_clear(wildcards):
     if config.RERUN_MODE:
-        return Path(f'{wildcards.dir}') / 'clear.done'
+        return Path(str('{wildcards.dir}')) / "clear.done"
```

Comment:
```suggestion
        return Path(wildcards.dir) / "clear.done"
```
wildcards.dir should already be a string. I think it can be simplified like this